### PR TITLE
meatballs

### PR DESCRIPTION
**Pull request by pontim**
#### Summary
Content "Add meatballs to the game"

#### Purpose of change
Add meatballs to the game

#### Describe the solution
Implemented JSON for raw and cooked meatballs

#### Testing
Spawned meatball and raw meatball, observed no issues with eating them

### CODE REVIEW

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -1385,5 +1385,48 @@
...
+    ],
+    "weight": "85g",
+    "color": "pink",
+    "spoils_in": "1 day",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 230,
+    "healthy": -1,
+    "parasites": 32,
+    "description": "A round, seasoned lump of meat, ready to fry",
```
Comment (casswedson) [inline data/json/items/comestibles/meat_dishes.json:1406]:
```suggestion
    "description": "A round, seasoned lump of meat, ready to fry.",
```

File: data/json/items/comestibles/meat_dishes.json [hunk 2]
```diff
@@ -1385,5 +1385,48 @@
...
+      { "type": "FLAG", "condition": "CANNIBALISM", "name": "manball" },
+      { "type": "FLAG", "condition": "STRICT_HUMANITARIANISM", "name": "murderball" },
+      { "type": "COMPONENT_ID", "condition": "mutant", "name": { "str_sp": "sinister %s" } }
+    ],
+    "copy-from": "meatball_raw",
+    "parasites": 0,
+    "healthy": 0,
+    "spoils_in": "4 days",
+    "price_postapoc": 300,
+    "description": "A seasoned, fried, round lump of meat. Just like mom used to make!",
```
Comment (casswedson) [inline data/json/items/comestibles/meat_dishes.json:1428]:
```suggestion
    "description": "A seasoned, fried, round lump of meat.  Just like mom used to make!",
```

**Comment by curstwist** [comment]
You could add recipes for spaghetti and meatballs and also meatball subs for better calorie returns per meal.

**Comment by pontim** [comment]
That's my plan. Also going to add "Deluxe" meatballs that include cheese and more calories

**Comment by ZhilkinSerg** [comment]
You need to add some recipe, otherwise they won't be available anywhere, but in debug menu.

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -7478,5 +7478,46 @@
...
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "time": "10 m",
+    "charges": 6,
+    "book_learn": [ [ "family_cookbook", 2 ], [ "cookbook_italian", 2 ] ],
+    "batch_time_factors": [ 50, 3 ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 1 } ],
+    "components": [
+      [ [ "meat_red", 2 ] ],
```
Comment (BrettDong) [inline data/json/recipes/recipe_food.json:7496]:
```suggestion
      [ [ "meat_red", 2, "LIST" ] ],
```

**Pull request merged**